### PR TITLE
Adds  dynamic RDS DBNAME parameter in secrets.

### PR DESCRIPTION
**Pull request by geetikabatra**
This branch would need rebasing once https://github.com/fabric8-analytics/fabric8-analytics-deployment/pull/23#pullrequestreview-107643116 is merged.

### CODE REVIEW

**Comment by geetikabatra** [comment]
@msrb and @tuxdna, PTAL. Please let me know what are your thoughts about adding RDS DBNAME as secret parameter.

**Review by humaton — CHANGES_REQUESTED** [review]
It does not really make sense to change DBNAME on user input.

File: openshift/secrets-template.yaml [hunk 1]
```diff
@@ -128,6 +128,12 @@ parameters:
   name: RDS_ENDPOINT
   value: "bm90LXNldA=="  # not-set
 
+- description: RDS database
```
Comment (tuxdna) [inline openshift/secrets-template.yaml:131]: Rename to `RDS database name`

File: openshift/secrets-template.yaml [hunk 2]
```diff
@@ -128,6 +128,12 @@ parameters:
...
+  displayName: RDS database
```
Comment (tuxdna) [inline openshift/secrets-template.yaml:132]: Rename to `RDS database name`

**Comment by tuxdna** [comment]
LGTM. I think this should work.

**Comment by tuxdna** [comment]
@GeetikaBatra on reading @humaton 's comment I want to understand why DB name is being parameter-ized ?

**Comment by geetikabatra** [comment]
So, the default value is set to `coreapi` here https://github.com/fabric8-analytics/fabric8-analytics-deployment/blob/master/openshift/secrets-template.yaml#L14
even if someone wants to change the database name in env.sh file. It still takes coreapi as default value.
So,I made this change to make dbname more dynamic. 
For a first timer, I faced a lot of struggle to deploy it on devcluster. This will make things a little easy in my opinion.

**Comment by humaton** [comment]
As we agreed with @GeetikaBatra this PR is not needed

**Comment by geetikabatra** [comment]
Thanks @humaton and @tuxdna, Closing this PR.

**Pull request closed without merging**